### PR TITLE
Update site as per mock

### PR DESCRIPTION
**Pull request by bhk31**
Hi @ArfatSalman.

As suggested at the time of presentation updated the site as per mock.

**Before**

before

**After**

after

### CODE REVIEW

**Review by AamirAlam — APPROVED** [review]

**Comment by AamirAlam** [comment]
Great work!  @bhk31 Looks good 🥇

**Comment by ArfatSalman** [comment]
Looks good! 

Also, ensure that you don't do redundant work from next time onwards. We need to move ahead quickly. I'm expecting your team's app to be one of highlights on 15th March for the final presentation. 😊 

If it's not complete by then, I won't be able to present it. So if you've done something, try to move ahead and complete the features. if you get time, you can re-work the design again. 

Do you guys understand? 

@bhk31 
@AamirAlam

File: src/component/carousel/Carousel.js [hunk 1]
```diff
@@ -2,32 +2,28 @@
 import { useState } from "react";
 import "./carousel.css";
 import { FormattedMessage } from "react-intl";
+import banner1 from "../../assets/carousel-images/1.png";
+import banner2 from "../../assets/carousel-images/2.png";
 // import { women } from "../../assets/carousel-images/women.jpg";
```
Comment (ArfatSalman) [inline src/component/carousel/Carousel.js:7]: Don't have  commented code in the PR.

**Comment by bhk31** [comment]
> Looks good!
> 
> Also, ensure that you don't do redundant work from next time onwards. We need to move ahead quickly. I'm expecting your team's app to be one of highlights on 15th March for the final presentation. 😊
> 
> If it's not complete by then, I won't be able to present it. So if you've done something, try to move ahead and complete the features. if you get time, you can re-work the design again.
> 
> Do you guys understand?
> 
> @bhk31
> @AamirAlam

Yes will take care of it.

File: src/component/carousel/Carousel.js (hunk 1 above)
Comment (bhk31) [reply src/component/carousel/Carousel.js:7]: Sure will update the PR

**Pull request merged**